examples/first-component/src/components/DrinkList/index.tsx

```diff
@@ -0,0 +1,35 @@
+interface DrinkProps {
+  name: string;
+}
+
+const Drink = ({ name }: DrinkProps) => {
+  const { part, caffeine, age } =
+    name === "tea"
+      ? { part: "leaf", caffeine: "15–70 mg/cup", age: "4,000+ years" }
```

Comment:
Pls do all challenges. The 3rd
`Challenge 3 of 3: Refactor a series of ? : to if and variables`

Reply:
Fixed